### DISCUSSION TITLE
[bug] Email button in Contact module widget behave differently for different user

### DISCUSSION THREAD

**Discussion by athpg** [YetiForceCompany/YetiForceCRM#15748] [General] [answered]
Hi everyone,

Not sure it is a bug or not. 
When I login as user A, in Contact module -> Emails widget (not tab) on the right -> Send email button, the button element is: \<button type="button" class="btn btn-sm btn-light sendMailBtn mailBtnActive" data-url="index.php?module=OSSMail&amp;view=Compose&amp;crmModule=Contacts&amp;crmRecord=194&amp;crmView=Detail&amp;type=new" data-module="Contacts" data-record="194" data-popup="1" title="Send email"\>\<span class="fas fa-envelope" title="Send email"\>\</span\>\</button\>
After I click the button, a roundcube client will open and has automatically filled the From: address.

However, when I login as user B, the same button, the Send email element changes to: \<a class="btn btn-sm btn-light" href="mailto:xxxxxx@gmail.com?subject=[C25] first name last name&amp;" title="Send email"\>\<span class="fas fa-envelope" title="Send email"\>\</span\>\</a\>
After I click the button, a browser mailto client (not roundcube) open and I couldn't enter anything in the From: address.

I am using the same browser, both user A and B has the same profile and in the same group, don't understand why the Send email button behave differently?

Thanks

**Comment by rskrzypczak** [comment, marked as answer]
Activate 'Internal mail composer' in user configuration.
![image](https://user-images.githubusercontent.com/10221999/155933469-c052d50d-2b70-4b1c-921c-c9a0d883c343.png)

**Reply by athpg** [reply]
Thanks, that is the option I missed. Problem solved.
